docs/5g-on-helium/cbrs-radios.mdx

```diff
@@ -97,27 +74,33 @@ change frequently.
 
 <div class="container" style={{ padding: 0, paddingTop: 0 }}>
   <div class="row">
-    <div class="col col--4" style={{ marginBottom: 0 }}>
-      <h3>All Good</h3>
+    <div class="col col--3" style={{ marginBottom: 0 }}>
       <img className="homeicon" src={useBaseUrl('img/5g-on-helium/sas-success-small.png')} />
       <p>
         The Small Cell Radio has been granted Spectrum Access by the SAS and is providing 5G
```

Comment:
Please have consistent terminology for Spectrum Access.

Reply:
this header has been pulled to avoid the naming collisions 👍